.github/workflows/scripts/node-badge.mjs

```diff
@@ -0,0 +1,50 @@
+"use strict";
+
+//! Calculate the percentage of games from node that we support
+// Expects node-gamedig checkout out in git root /node-gamedig
+// Expects the generic example to output a list of game IDs when no arguments are provided
+
+import process from "node:process";
+import { closeSync, openSync, writeSync } from "node:fs";
+import { spawnSync } from "node:child_process";
+
+function setOutput(key, value) {
+    const file = openSync(process.env.GITHUB_OUTPUT, "a");
+    writeSync(file, `${key}=${value}\n`);
+    closeSync(file);
+}
+
+// Get node IDs
+
+import { games } from "../../../node-gamedig/lib/games.js";
```

Comment:
[node-GameDig#411](https://github.com/gamedig/node-gamedig/pull/411) changed so that `games` are exported on the main lib file, so no need for this.

Reply:
It's imported like this because of the sparse checkout: https://github.com/gamedig/rust-gamedig/pull/156/files#diff-7cea706dd266dd114c5f9cb504bf263cb7f3eb5504dca1e7e07b319c6193746eR29-R31

which only creates the lib/games.js file.

Although this probably doesn't save that many resources so probably worth just using index.js to save breakages if the games.js file moves.

Reply:
We could leave it like this, a comment to mention on why we do the import like this would be good to have tho.

Reply:
I did change it to this, but actually that requires that we install the dependencies so I will change it back.